server/src/main/java/org/gluu/oxtrust/util/jsf/JsfFunctions.java

```diff
@@ -154,5 +160,14 @@ public static String trimToLength(String str, int maxLength) {
 
 		return str.substring(0, maxLength) + "...";
 	}
+	
+	public static String convertLdapTimeToDisplay(String timeStamp) {
+		if ((timeStamp == null) || (timeStamp.length() == 0)) {
+			return timeStamp;
+		}
+		
+		return CdiUtil.bean(LdapEntryManager.class).decodeGeneralizedTime(timeStamp).toGMTString();
```

Comment:
Did you added this because ldapEntryManager is null? Can you remove injection in this case.